exporter/stats/stackdriver/stackdriver.go

```diff
@@ -119,7 +117,25 @@ func (e *Exporter) Export(vd *stats.ViewData) {
 	if len(vd.Rows) == 0 {
 		return
 	}
-	e.bundler.Add(vd, 1)
+	err := e.bundler.Add(vd, 1)
+	switch err {
+	case nil:
+		return
+	case bundler.ErrOversizedItem:
+		go e.handleUpload(vd)
+	case bundler.ErrOverflow:
+		e.onError(errors.New("OpenCensus stats/Stackdriver exporter: failed to upload View: buffer full"))
```

Comment:
"failed to upload: buffer full"

onError will automatically add a Stackdriver exporter to the log.

Reply:
Aye aye, thanks. Let me update that.